packages/core/src/components/modal-header/modal-header.scss

```diff
@@ -11,13 +11,12 @@
 :host {
   @include ix-component;
   display: flex;
-  padding: 0.5rem;
+  padding: 0.5rem 0 0.5rem 0.5rem;
   align-items: center;
   align-self: stretch;
 
   .modal-close {
     margin-left: auto;
```

Comment:
```suggestion
    margin-inline-start: 1rem;
```

Reply:
Why do we need this? If i apply this change the icon would be placed next to the text?
![image](https://github.com/user-attachments/assets/f4fac710-52f3-4a2a-9f0f-1bc004951997)